client/src/App.js

```diff
@@ -7,22 +7,23 @@ import DetailContainer from './Components/Organization/OrganizationDetail/Detail
 import SignIn from './Components/Authentication/Signin'
 import SignUp from './Components/Authentication/Signup'
 import ProjectDetails from './Components/Organization/OrganizationDetail/ProjectDetails/ProjectDetailsContainer'
-
+import SelectedProposals from './Components/Dashboard/Proposals/SelectedProposals';
+import YourProposals from './Components/Dashboard/Proposals/YourProposals';
 class App extends Component {
   render() {
     return (
       <BrowserRouter>
-	  <Navbar />
+	    <Navbar />
```

Comment:
Navbar should not be outside the app, please correct indention for this as well.

Reply:
Thanks for suggesting changes. I have updated the changes. Please have a look.